app/views/groups/_form_fields.html.erb

```diff
@@ -1,31 +1,81 @@
 <% invalid_name = group.errors.include?(:name) %>
 <div class="form-field <%= 'invalid' if invalid_name %>">
   <%= form.label :name %>
-  <%= form.text_field :name, data: { "slugify-target": "name", action: "input->slugify#nameChanged" }, placeholder: t(:'groups.create.name_placeholder'), required: true %>
-  <%= render 'shared/form/field_errors', errors: group.errors.full_messages_for(:name) %>
+  <%= form.text_field :name,
+                  data: {
+                    "slugify-target": "name",
+                    action: "input->slugify#nameChanged"
+                  },
+                  placeholder: t(:"groups.create.name_placeholder"),
+                  required: true %>
+  <%= render "shared/form/field_errors", errors: group.errors.full_messages_for(:name) %>
 </div>
 
 <% if group.parent_id.present? %>
   <div class="grid grid-cols-2 gap-4">
     <div class="form-field">
       <%= form.label :parent_id %>
       <div class="flex whitespace-nowrap items-center">
-        <div class="inline-flex items-center px-3 py-2.5 text-sm text-gray-600 bg-gray-200 dark:bg-gray-800/50 dark:text-gray-200 border border-r-0 border-gray-300 rounded-l-lg dark:bg-gray-600 dark:text-gray-400 dark:border-gray-600">
+        <div
+          class="
+            inline-flex
+            items-center
+            px-3
+            py-2.5
+            text-sm
+            text-gray-600
+            bg-gray-200
+            dark:bg-gray-800/50
+            dark:text-gray-200
+            border
+            border-r-0
+            border-gray-300
+            rounded-l-lg
+            dark:bg-gray-600
+            dark:text-gray-400
+            dark:border-gray-600
+          "
+        >
           <%= root_url %>
         </div>
         <%= form.collection_select(
-              :parent_id,
-              Namespace.where(type: [Group.sti_name, Namespaces::UserNamespace.sti_name], owner: current_user),
-              :id,
-              :full_path, {},
-              { :class => "prefixed" }
-            ) %>
+          :parent_id,
+          Namespace
```

Comment:
Can you change this to use a query in the controller, we really shouldn't be performing queries in the views. I think this would be a perfect use case for an additional scope (i.e. :manageable).  See https://actionpolicy.evilmartians.io/#/scoping?id=define-scopes

Reply:
Updated to use a scope in d77f8dac7a63b832a3c1d00b00315f2eaa342805